composer.json

```diff
@@ -11,19 +11,25 @@
     "license": "MIT",
     "require": {
         "php": "^7.4",
-        "pestphp/pest": "^0.2",
+        "pestphp/pest": "dev-master",
```

Comment:
Required because https://github.com/pestphp/pest/pull/109 isn't released yet.
Will change this as soon as `v0.2.3` is released.